src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/DynamicDebuggerProxy.cs

```diff
@@ -0,0 +1,482 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Dynamic;
+using System.Linq.Expressions;
+using System.Runtime.CompilerServices;
+using System.Runtime.Serialization;
+
+namespace Microsoft.CSharp.RuntimeBinder
+{
+    [Serializable]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    internal sealed class DynamicBindingFailedException : Exception
+    {
+        public DynamicBindingFailedException()
+            : base()
+        {
+        }
+
+        private DynamicBindingFailedException(SerializationInfo info, StreamingContext context) 
+            : base(info, context) 
+        { 
+        }
+    }
+
+    internal sealed class GetMemberValueBinder : GetMemberBinder
+    {
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")]
```

Comment:
Do we run this rule in corefx?  Wondering if this is still needed.

Reply:
I actually wasn't aware we respect any CodeAnalysis attributes in CoreFX. Do we? I think we've generally deleted them.